### DISCUSSION TITLE
Regex Expression For Column Header Annotations?

### DISCUSSION THREAD

**Discussion by elderdo** [ozlerhakan/poiji#262] [Ideas]
Have you ever considered allowing regular expressions for column header annotations?  That would provide a lot of flexibility in matching with Headers that may not match exactly, but are close.

**Comment by ozlerhakan** [comment]
Hi @elderdo ,
 
It would be so preferable feature! Do you want to contribute to Poiji?

**Reply by aerfus** [reply]
Hello @ozlerhakan ,
could you confirm the feature is not implemented yet.

If so, I would like to contribute and implement in a similar way to
https://opencsv.sourceforge.net/#multivaluedmap_based_bean_fields_many_to_one_mappings

Please let me know your concerns.
Thank you in advance,

**Reply by ozlerhakan** [reply]
Hi @aerfus , please go ahead, we haven't had regex support in Poiji.
